_includes/fsm.html

```diff
@@ -3,7 +3,7 @@
 
     <script src="../assets/js/ModuleCheckpoints.js"></script>
     <script
-            src="http://ajax.aspnetcdn.com/ajax/jQuery/jquery-1.10.2.min.js"></script>
+            src="https://ajax.aspnetcdn.com/ajax/jQuery/jquery-3.4.1.min.js"></script>
```

Comment:
@mohit-003  Can you do something so that you don't have to mention http or https explicitly?

Reply:
> @mohit-003 Can you do something so that you don't have to mention http or https explicitly?

Yes done!